helm/values-sandbox.yaml

```diff
@@ -0,0 +1,203 @@
+
+#
+# FLYTEADMIN
+#
+
+flyteadmin: {} # use default config
+
+#
+# DATACATALOG
+#
+
+datacatalog: {}
+
+#
+# FLYTEPROPELLER
+#
+
+flytepropeller: {}
+
+#
+# FLYTECONSOLE
+#
+
+flyteconsole: {}
+
+#
+# REDIS
+#
+
+redis:
+  enabled: false
+
+#
+# POSTGRES
+#
+
+postgres:
+  image:
+    # Override postgres image to use alpine based (rather smaller) docker image
+    tag: "10.16-alpine"
+
+#
+# MINIO
+#
+
+minio: {}
+
+#
+# CONTOUR
+#
+
+contour:
+  contour: {}
+  envoy:
+    service:
+      type: NodePort
+      ports:
+        http: 80
+      nodePorts:
+        http: 30081
+
+#
+# SPARKOPERATOR
+#
+
+sparkoperator:
+  enabled: false
+
+#
+# PYTORCHOPERATOR
+#
+
+pytorchoperator:
+  enabled: false
+
+#
+# KUBERNETES DASHBOARD
+#
+kubernetes-dashboard:
+  enabled: true
+  extraArgs:
+   - --enable-skip-login
+   - --enable-insecure-login
+   - --disable-settings-authorizer
+  protocolHttp: true
+  service:
+    type: NodePort
+    externalPort: 30082
+
+#
+# COMMON
+#
+
+common:
+  databaseSecret: {}
+  ingress:
+    # enable HMR route to flyteconsole for frontend development.
+    webpackHMR: true
+  flyteNamespaceTemplate: {}
+
+#
+# CONFIGMAPS
+#
+
+configmap:
+
+  db:
+    database:
+      port: 5432
+      username: postgres
+      host: postgres
+      dbname: flyte_development
+
+  storage:
```

Comment:
I would like to change this
Have a parameter like
storage-kind: s3 | gcs | minio | custom

If custom then you provide the entire configuration, else the config is automatic?

Reply:
This is now done in https://github.com/flyteorg/flyte/commit/151446d39359209a04bb28aeaa5cd448b4f8f9e0 (in the new internal branch).